build.sbt

```diff
@@ -58,6 +58,10 @@ lazy val commonDependencies = Seq(
 )
 
 lazy val root = (project in file("."))
+  .settings(
```

Comment:
Will this cause problems because there is already a support-frontend project beneath the root? Doesn't this make the sbt command `project support-frontend` ambiguous?

Reply:
You're right, fixed.

Reply:
Can we just leave it as root? Less typing 😃

Reply:
@rupertbates Would be nice it if's not confusing in InteliJ, every other project is properly named while this one is called root, so it's not consistent and can clash, since if you don't put a module name SBT will use variable name.